web_src/js/features/repo-home.ts

```diff
@@ -7,141 +7,144 @@ import {fomanticQuery} from '../modules/fomantic/base.ts';
 const {appSubUrl} = window.config;
 
 export function initRepoTopicBar() {
-  const mgrBtn = document.querySelector('#manage_topic');
-  if (!mgrBtn) return;
+  const mgrBtns = document.querySelectorAll<HTMLButtonElement>('#manage_topic');
+  if (mgrBtns.length === 0) return;
 
-  const editDiv = document.querySelector('#topic_edit');
-  const viewDiv = document.querySelector('#repo-topics');
-  const topicDropdown = editDiv.querySelector('.ui.dropdown');
-  let lastErrorToast;
+  for (const mgrBtn of mgrBtns) {
+    const editDiv = mgrBtn.parentNode.querySelector('#topic_edit');
+    const viewDiv = mgrBtn.parentNode.querySelector('#repo-topics');
+    const topicDropdown = editDiv.querySelector('.ui.dropdown');
+    let lastErrorToast;
 
-  mgrBtn.addEventListener('click', () => {
-    hideElem(viewDiv);
-    showElem(editDiv);
-    topicDropdown.querySelector('input.search').focus();
-  });
+    mgrBtn.addEventListener('click', () => {
+      hideElem(viewDiv);
+      showElem(editDiv);
+      topicDropdown.querySelector<HTMLInputElement>('input.search').focus();
+    });
 
-  document.querySelector('#cancel_topic_edit').addEventListener('click', () => {
-    lastErrorToast?.hideToast();
-    hideElem(editDiv);
-    showElem(viewDiv);
-    mgrBtn.focus();
-  });
+    mgrBtn.parentNode.querySelector('#cancel_topic_edit').addEventListener('click', () => {
+      lastErrorToast?.hideToast();
+      hideElem(editDiv);
+      showElem(viewDiv);
+      mgrBtn.focus();
+    });
 
-  document.querySelector('#save_topic').addEventListener('click', async (e) => {
-    lastErrorToast?.hideToast();
-    const topics = editDiv.querySelector('input[name=topics]').value;
+    mgrBtn.parentNode.querySelector('#save_topic').addEventListener('click', async (e: MouseEvent & {target: HTMLAnchorElement}) => {
```

Comment:
No, `save_topic` isn't an "anchor"

Reply:
Done in 0a2d81c6ee7c62d9caa45e52e7ab305dc10409aa